keyboards/splitography/rules.mk

```diff
@@ -0,0 +1,40 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
```

Comment:
I removed `OPT_DEFS += -DBOOTLOADER_SIZE=512` and put this line instead. I'm not 100% sure but it seems to work fine on my keyboard.